LinearRegression/LinearRegressionBackend/Program.cs

```diff
@@ -55,12 +56,14 @@ static async Task<int> Main(string[] args)
             trainCommand.AddGlobalOption(layersOption);
 
             var activationFuncOption =
-                new Option<string>(
+                new Option<string[]>(
                     name: "--activationfunc",
                     description:
-                        "The activation function used by the neural network")
+                        "The activation functions used by each layer.")
                 {
                     IsRequired = true,
+                    Arity = ArgumentArity.OneOrMore,
```

Comment:
Wow I do not even know what is 'arity' means. 

> Arity is the number of arguments or operands taken by a function, operation or relation in logic, mathematics, and computer science

I learned a new thing today! 😄